src/tsconfig.json

```diff
@@ -24,6 +25,13 @@
       "es2018",
       "dom"
     ],
-    "preserveSymlinks": true
+    "preserveSymlinks": true,
+    "paths": {
+      "@acme/extensions": ["frontend/packages/acme_extensions/src/public-api.ts"],
+      "@stratos/core": ["frontend/packages/core/src/public-api.ts"],
+      "@stratos/shared": ["frontend/packages/shared/src/public-api.ts"],
+      "@stratos/store": ["frontend/packages/shared/src/public-api.ts"],
```

Comment:
points to `shared`, also the `public-api` in store points to invalid export `./lib/store.module`

Reply:
Left store in - this is where we can build out the public api